manifests/0000_30_cluster-baremetal-operator_03_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-baremetal-operator
+  namespace: cluster-baremetal-operator
```

Comment:
Are we sure we want to use this namespace? Why not using openshift-machine-api, I wonder? If we do want to use the cbo namespace, we should have a manifest to create it as well before we create the service account.

Reply:
Thanks, my mistake, I'll fix it!